frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -980,6 +981,30 @@ class Worksheet extends React.Component {
                         this.discardChanges();
                     }.bind(this),
                 });
+                editor.commands.addCommand({
+                    name: 'comment',
+                    bindKey: { win: 'Ctrl+/', mac: 'cmd+/' },
+                    exec: function() {
+                        let selectionRange = editor.getSelectionRange();
+                        selectionRange.start.column = 0;
+                        let content = editor.session.getTextRange(selectionRange);
+                        let splitLines = content.split('\n');
+                        let commentAll = true;
+                        if (splitLines[0].startsWith('//')) {
```

Comment:
Our rule will be: if the first selected line is in commented status, we uncomment every line, else we comment every line